cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -420,6 +424,45 @@ impl CoreEnvironment<ReadOnly> {
         })
     }
 
+    fn get_install_ids_to_uninstall(
+        manifest: &TypedManifest,
```

Comment:
Note to self: this can take a `&TypedManifestCatalog` instead since we're about to no longer allow locking v0  (`pkgdb`) manifests, which removes the need for `if let ...`